### PR TITLE
Add Equal to some modules

### PR DESCRIPTION
**Pull request by duzhen1996**

### CODE REVIEW

File: smart-common/src/test/java/org/smartdata/model/TestActionInfo.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package org.smartdata.model;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Random;
+
+/**
```
Comment (qiyuangong) [inline smart-common/src/test/java/org/smartdata/model/TestActionInfo.java]: Remove this comment.

File: smart-common/src/main/java/org/smartdata/model/ActionInfo.java [hunk 2]
```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```
Comment (qiyuangong) [inline smart-common/src/main/java/org/smartdata/model/ActionInfo.java]: Do not change this comment, when using code format.

**Comment by huafengw** [comment]
Please use IDE's code generate tool to generate corresponding `equals` and `hashCode` function.

File: smart-common/src/test/java/org/smartdata/model/TestActionInfo.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
+package org.smartdata.model;
```
Comment (huafengw) [inline smart-common/src/test/java/org/smartdata/model/TestActionInfo.java]: License header

File: smart-common/src/test/java/org/smartdata/model/TestCmdletInfo.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
+package org.smartdata.model;
```
Comment (huafengw) [inline smart-common/src/test/java/org/smartdata/model/TestCmdletInfo.java]: License header

File: smart-common/src/main/java/org/smartdata/model/ActionInfo.java [hunk 5]
```diff
@@ -99,7 +128,8 @@ public String getArgsJsonString() {
   public void setArgsFromJsonString(String jsonArgs) {
     Gson gson = new Gson();
     args = gson.fromJson(jsonArgs,
-        new TypeToken<Map<String, String>>(){}.getType());
+            new TypeToken<Map<String, String>>() {
```
Comment (qiyuangong) [inline smart-common/src/main/java/org/smartdata/model/ActionInfo.java]: Use the same code style in the project.

File: smart-common/src/main/java/org/smartdata/model/ActionInfo.java [hunk 6]
```diff
@@ -238,8 +268,8 @@ public static Builder create() {
     }
 
     public ActionInfo build() {
-      return new ActionInfo(actionId,cmdletId,actionName, args, result,
-          log, successful, createTime, finished, finishTime, progress);
+      return new ActionInfo(actionId, cmdletId, actionName, args, result,
```
Comment (qiyuangong) [inline smart-common/src/main/java/org/smartdata/model/ActionInfo.java]: Code style.

File: smart-common/src/main/java/org/smartdata/model/CmdletInfo.java [hunk 7]
```diff
@@ -35,8 +35,44 @@ public CmdletInfo() {
...
+            || (parameters != null ? !parameters.equals(cmdletInfo.parameters) : cmdletInfo.parameters != null)) {
+      return false;
+    }
+
+    return true;
+  }
+
   public CmdletInfo(long cid, long rid, CmdletState state,
-      String parameters, long generateTime, long stateChangedTime) {
+                    String parameters, long generateTime, long stateChangedTime) {
```
Comment (qiyuangong) [inline smart-common/src/main/java/org/smartdata/model/CmdletInfo.java]: Code style.

**Pull request merged**